src/components/notification/notification_event_read_button.tsx

```diff
@@ -50,29 +50,40 @@ export const EuiNotificationEventReadButton: FunctionComponent<EuiNotificationEv
     'euiNotificationEventReadButton--isRead': isRead,
   });
 
-  const markAsRead = useEuiI18n(
-    'euiNotificationEventReadButton.markAsRead',
-    'Mark {eventName} as read.',
+  const markAsReadAria = useEuiI18n(
+    'euiNotificationEventReadButton.markAsReadAria',
+    'Mark {eventName} as read',
     {
       eventName,
     }
   );
 
-  const markAsUnread = useEuiI18n(
-    'euiNotificationEventReadButton.markAsUnread',
-    'Mark {eventName} as unread.',
+  const markAsUnreadAria = useEuiI18n(
+    'euiNotificationEventReadButton.markAsUnreadAria',
+    'Mark {eventName} as unread',
     {
       eventName,
     }
   );
 
-  const buttonLabel = !isRead ? markAsRead : markAsUnread;
+  const markAsRead = useEuiI18n(
+    'euiNotificationEventReadButton.markAsRead',
+    'Mark as read'
+  );
+
+  const markAsUnread = useEuiI18n(
+    'euiNotificationEventReadButton.markAsUnread',
+    'Mark as unread'
+  );
+
+  const buttonAriaLabel = !isRead ? markAsReadAria : markAsUnreadAria;
+  const buttonTitle = !isRead ? markAsRead : markAsUnread;
 
   return (
     <EuiButtonIcon
       iconType="dot"
-      aria-label={buttonLabel}
-      title={buttonLabel}
+      aria-label={buttonAriaLabel}
+      title={buttonTitle}
```

Comment:
This is what I meant about separating the title vs aria-label. So the aria-label is more descriptive for screen-readers to associate the button with the title, but for mouse/visual users, the title is redundant in the tooltip, so this is just more succinct.

Reply:
I see. Thanks!